source/includes/markdown/_17-audit-log.html.md

```diff
@@ -38,26 +38,27 @@ All user events operate on the _User_ resource type.
  
 ### Admin Settings
  
-All admin settings events operate on the _Workspace_ resource type.
- 
-| Event Type | Description |
-|------------|---------------|
-| workspace_google_sso_settings_changed            | The workspace's Google SSO settings were changed. |
-| workspace_saml_settings_changed                  | The workspace's SAML settings were changed. |
-| workspace_saml_url_changed                       | The workspace's SAML url was changed. |
-| workspace_password_requirements_changed          | The workspace's password strength requirements were changed. |
-| workspace_force_password_reset                   | All users in the workspace were forced to reset their password. |
-| workspace_guest_invite_permissions_changed       | The workspace’s guest invite permissions were changed. |
-| workspace_file_attachment_options_changed        | File attachment options were enabled or disabled for the workspace. |
-| workspace_default_team_privacy_settings_changed  | The workspace's default team privacy settings were changed. |
-| workspace_wide_reporting_enabled                 | Workspace wide reporting was enabled. |
-| workspace_wide_reporting_disabled                | Workspace wide reporting was disabled. |
-| workspace_associated_email_domain_added          | An email domain was added to the workspace. |
-| workspace_associated_email_domain_removed        | An email domain was removed from the workspace. |
-| workspace_require_two_factor_auth_enabled        | Two factor authentication was set as required for the workspace. |
-| workspace_require_two_factor_auth_disabled       | Two factor authentication was set as not required for the workspace. |
-| workspace_view_links_enabled                     | Read-only link sharing was enabled for the workspace. |
-| workspace_view_links_disabled                    | Read-only link sharing was disabled for the workspace. |
+| Event Type | Resource Type | Description |
```

Comment:
@androse , do you mind switching this back to a two-column table? With the way that our docs are designed, the changes look a bit cramped:

![Screen Shot 2022-01-21 at 4 10 17 PM](https://user-images.githubusercontent.com/82971401/150615582-1e830670-817e-48f5-8da7-3ada2d25ee0c.png)